features/step_definitions/scc.rb

```diff
@@ -64,6 +64,9 @@
 
 Given /^SCC #{QUOTED} is (added to|removed from) the #{QUOTED} (user|group|service account)( without teardown)?$/ do |scc, op, which, type, no_teardown|
   ensure_admin_tagged
+  if no_teardown
+    ensure_upgrade_prepare_tagged
```

Comment:
I think method should be called `ensure_upgrade_tagged` and also allow the upgrade check tag. In both - upgrade check and upgrade prepare we do not want clean-up. Otherwise after one attempt of check, the data from upgrade prepare will be gone.

In this case though things may make sense to be left as you did them. Because within upgrade check we shouldn't change SCC.

WDYT? Perhaps lets leave as you have it and wi will see if upgrade-check needs to be added in the future.

Reply:
@akostadinov   Add in https://github.com/openshift/verification-tests/pull/1448